### PR TITLE
Add Wearables Client to dump collections

### PR DESCRIPTION
**Pull request by mikhail-dcl**
- Added an additional tab to convert wearables collection, by default it points to base wearables 
- Added the flag "Place on the scene"

### CODE REVIEW

File: asset-bundle-converter/Assets/AssetBundleConverter/AssetBundleConverter.cs [hunk 1]
```diff
@@ -84,7 +84,7 @@ public AssetBundleConverter(Environment env, ClientSettings settings)
 
             log.verboseEnabled = true;
 
-            ShaderGraphMaterialGenerator.k_ShaderPathPrefix = "Assets/git-submodules/glTFast/Runtime/Shader/";
+            // ShaderGraphMaterialGenerator.k_ShaderPathPrefix = "Assets/git-submodules/glTFast/Runtime/Shader/";
```
Comment (Kinerius) [inline asset-bundle-converter/Assets/AssetBundleConverter/AssetBundleConverter.cs:87]: If you pull you should be able to revert this change

Comment (mikhail-dcl) [reply asset-bundle-converter/Assets/AssetBundleConverter/AssetBundleConverter.cs:87]: ah yeah, you're right, I forgot
